1-bootstrap/README.md

```diff
@@ -22,6 +22,11 @@ Each pipeline has the following associated resources:
 
 ## Usage
 
+### Pre-requisites
+
+- Google Cloud SDK version greater than 487.0.0
+- Terraform version greater than 1.6
```

Comment:
technically this particular step only required TF v1.5

Reply:
yes, you are right

do we move this pre-requisites to another section?

my objective is for the user to not have issues later on when deploying fleetscope

Reply:
My initial thought is it would be more fitting under the root (`/`) README?

Reply:
sounds about right, I'll be updating this